### PR TITLE
Add some links and fix typos

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,10 +161,9 @@ Follow the instruction on <https://rustup.rs/> to install rust and cargo.
 
 Then try running `npm install` again.
 
-Make sure that your nodejs version is `16.0.0` or newer.
+Make sure that your nodejs version is `18.0.0` or newer.
 
-If you still get errors look at the instructions in the deltchat-node and
```
Comment (ralphtheninja) [inline README.md:166]: Typo + add link

File: README.md [hunk 2]
```diff
@@ -161,10 +161,9 @@ Follow the instruction on <https://rustup.rs/> to install rust and cargo.
...
-deltachat-rust-core README files to set things up **or write an issue**.
```
Comment (ralphtheninja) [inline README.md:167]: It's really deltachat-core-rust + add link

File: README.md [hunk 3]
```diff
@@ -161,10 +161,9 @@ Follow the instruction on <https://rustup.rs/> to install rust and cargo.
 
 Then try running `npm install` again.
 
-Make sure that your nodejs version is `16.0.0` or newer.
```
Comment (ralphtheninja) [inline README.md:164]: Node v16 has reached end of life

File: README.md (hunk 2 above)
Comment (ralphtheninja) [reply README.md:167]: Add link to creating a new issue

File: README.md [hunk 4]
```diff
@@ -152,7 +152,7 @@ For development with local deltachat-core read the [docs](docs/UPDATE_CORE.md)
 
 ### Troubleshooting <a id="troubleshooting"></a>
 
-This module builds on top of `deltachat-core-rust`, which in turn has external dependencies. Instructions below assumes a Linux system (e.g. Ubuntu 18.10).
+This module builds on top of [`deltachat-core-rust`](https://github.com/deltachat/deltachat-core-rust), which in turn has external dependencies. Instructions below assumes a Linux system (e.g. Ubuntu 18.10).
```
Comment (dotlambda) [inline README.md:155]:
```suggestion
This module builds on top of [`deltachat-core-rust`](https://github.com/deltachat/deltachat-core-rust), which in turn has external dependencies. The instructions below assume a Linux system (e.g. Ubuntu 18.10).
```

File: README.md [hunk 5]
```diff
@@ -161,10 +161,9 @@ Follow the instruction on <https://rustup.rs/> to install rust and cargo.
 
 Then try running `npm install` again.
 
-Make sure that your nodejs version is `16.0.0` or newer.
+Make sure that your nodejs version is `18.0.0` or newer.
 
-If you still get errors look at the instructions in the deltchat-node and
-deltachat-rust-core README files to set things up **or write an issue**.
+If you still get errors look at the instructions in the [`deltachat-node`](https://github.com/deltachat/deltachat-node) and [`deltachat-core-rust`](https://github.com/deltachat/deltachat-core-rust) README files to set things up [`or write an issue`](https://github.com/deltachat/deltachat-desktop/issues/new/choose).
```
Comment (dotlambda) [inline README.md:166]: deltachat-node was moved into the core repo

File: README.md (hunk 3 above)
Comment (Simon-Laux) [reply README.md:164]: electron 22 (which we use) still uses node 16: https://www.electronjs.org/blog/electron-22-0

Comment (Simon-Laux) [reply README.md:164]: so I think it's better to do that later as part of #3381 also to really change it we need to update all occurrences and ci files.

**Review by Simon-Laux — CHANGES_REQUESTED** [review]
Looks good besides the node upgrade details above.

File: README.md (hunk 3 above)
Comment (ralphtheninja) [reply README.md:164]: > electron 22 (which we use) still uses node 16: https://www.electronjs.org/blog/electron-22-0

This is really unrelated imo

> so I think it's better to do that later as part of #3381 also to really change it we need to update all occurrences and ci files.

Much better argument! I'll tweak this and make a separate PR for all node 16 thingies :smile:

Comment (Simon-Laux) [reply README.md:164]: > This is really unrelated imo

not really, the version electron ships is the one the code actually runs with.

**Review by Simon-Laux — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (ralphtheninja) [reply README.md:164]: > > This is really unrelated imo
> 
> not really, the version electron ships is the one the code actually runs with.

So then it doesn't matter what node version you're running.

**Pull request merged**